### PR TITLE
Add alias for Service 'StartType'

### PR DESCRIPTION
**Pull request by NeoBeum**
# PR Summary
Add alias for Set-Service -StartType

## PR Context

Current Get;Set; Service parameters are: "StartMode", "StartupType", "ST", "SM".
Get-Service displays "StartType", StartType is a member for NativeMethods ServiceController.InputObject.
Addition of the alias "StartType" helps scripting for Get;Set; as variable name does not need to be changed.

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [X] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [X] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [X] Issue filed: https://github.com/MicrosoftDocs/PowerShell-Docs/issues/4562
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs [hunk 1]
```diff
@@ -1543,13 +1543,14 @@ public string Description
 
         /// <summary>
         /// The following is the definition of the input parameter "StartupType".
+        /// "StartType" is NativeMethod Get;Set; ServiceController.InputObject.StartType
```
Comment (adityapatwardhan) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs]: Please reword the comment, it is a bit difficult to understand on the first read.

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs (hunk 1 above)
Comment (NeoBeum) [reply src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs]: @adityapatwardhan 
Will this be ok?

"Set-Service sets ServiceController.InputObject.StartType"

Comment (anmenaga) [reply src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs]: @adityapatwardhan . This PR needs your approval.

**Comment by anmenaga** [comment]
Failures on CI-macos were unrelated and were run long time ago, so I restarted the CI-macos.

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**